pages/about.html

```diff
@@ -1,12 +1,53 @@
 <!DOCTYPE html>
 <html lang="en">
-<head>
-  <meta charset="UTF-8">
-  <meta http-equiv="X-UA-Compatible" content="IE=edge">
-  <meta name="viewport" content="width=device-width, initial-scale=1.0">
-  <title>Document</title>
-</head>
+  <head>
+    <meta charset="UTF-8">
+    <meta name="viewport" content="width=device-width, initial-scale=1.0">
+    <title>Atefeh Mirzaei, about</title>
+    <link rel="preconnect" href="https://fonts.gstatic.com" crossorigin>
+    <link href="https://fonts.googleapis.com/css2?family=Caveat&display=swap" rel="stylesheet">
+    <link rel="stylesheet" href="/styles/style.css">
+  </head>
 <body>
-  
+  <header>
+    <section class="content-wrap">
+      <h2>Atefeh Mirzaei</h2>
+        <ul class="nav-bar">
+          <li><a href="/pages/index.html">Home</a></li>
+          <li><a href="/pages/about.html">About</a></li>
+          <li><a href="/pages/portfolio.html">Portfolio</a></li>
+        </ul>
+    </section>
+  </header>
+  <!-- ******************  EDUCATION & CERTIFICATIONS ****************** -->
+  <section class="education">
+    <div class="content-wrap item-details">
+      <h2>Education</h2>
+      <!-- Copy this whole <section> block to add more schools. -->
+      <section>
+        <h3>Ada developers Academy</h3>
+        <p>Software development , Sept 2022 - July 2023 </p>
+        <h3>Applied Science University , Amman Jordan</h3>
+        <p>Communication and Electronic, 2000 - 2005</p>
+        <h3>Al-Amerat high school, Muscat Oman</h3>
+        <p>high school, graduated in 2000 </p>  
+      </section>
+    <!-- End of School block. -->
+  </div>
+  </section>
+</div>
+<!-- End of Job block. -->
+</section>
```

Comment:
A couple of stray end tags here. `</div>` on line 38 and this `</section>` tag don't have a matching opening tag.

Make sure you're using the HTML Validator to catch things like this. https://validator.w3.org